coverage/clover.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<coverage generated="1651251687651" clover="3.2.0">
+  <project timestamp="1651251687652" name="All files">
+    <metrics statements="16" coveredstatements="9" conditionals="2" coveredconditionals="1" methods="13" coveredmethods="7" elements="31" coveredelements="17" complexity="0" loc="16" ncloc="16" packages="2" files="4" classes="4"/>
+    <package name="components">
+      <metrics statements="3" coveredstatements="0" conditionals="0" coveredconditionals="0" methods="3" coveredmethods="0"/>
+      <file name="BurrowCards.js" path="/Users/learnacademy/Desktop/Rabbit-Hole/app/javascript/components/components/BurrowCards.js">
+        <metrics statements="3" coveredstatements="0" conditionals="0" coveredconditionals="0" methods="3" coveredmethods="0"/>
+        <line num="15" count="0" type="stmt"/>
+        <line num="19" count="0" type="stmt"/>
+        <line num="22" count="0" type="stmt"/>
+      </file>
+    </package>
+    <package name="pages">
+      <metrics statements="13" coveredstatements="9" conditionals="2" coveredconditionals="1" methods="10" coveredmethods="7"/>
+      <file name="AboutUs.js" path="/Users/learnacademy/Desktop/Rabbit-Hole/app/javascript/components/pages/AboutUs.js">
+        <metrics statements="2" coveredstatements="2" conditionals="0" coveredconditionals="0" methods="2" coveredmethods="2"/>
+        <line num="18" count="1" type="stmt"/>
+        <line num="21" count="1" type="stmt"/>
+      </file>
+      <file name="Home.js" path="/Users/learnacademy/Desktop/Rabbit-Hole/app/javascript/components/pages/Home.js">
+        <metrics statements="9" coveredstatements="5" conditionals="2" coveredconditionals="1" methods="6" coveredmethods="3"/>
+        <line num="6" count="1" type="stmt"/>
+        <line num="7" count="1" type="stmt"/>
+        <line num="15" count="1" type="stmt"/>
+        <line num="16" count="0" type="stmt"/>
+        <line num="18" count="0" type="stmt"/>
+        <line num="23" count="0" type="stmt"/>
+        <line num="27" count="1" type="stmt"/>
+        <line num="28" count="1" type="cond" truecount="1" falsecount="1"/>
+        <line num="29" count="0" type="stmt"/>
+      </file>
+      <file name="NotFound.js" path="/Users/learnacademy/Desktop/Rabbit-Hole/app/javascript/components/pages/NotFound.js">
+        <metrics statements="2" coveredstatements="2" conditionals="0" coveredconditionals="0" methods="2" coveredmethods="2"/>
+        <line num="12" count="1" type="stmt"/>
+        <line num="15" count="1" type="stmt"/>
+      </file>
+    </package>
+  </project>
+</coverage>
```

Comment:
Add the folder app/coverage to you gitignore. And remove this folder from your app.